src/codewhisperer/util/showSsoPrompt.ts

```diff
@@ -15,27 +15,14 @@ import { ToolkitError } from '../../shared/errors'
 import { createCommonButtons } from '../../shared/ui/buttons'
 import { telemetry } from '../../shared/telemetry/telemetry'
 import { isCloud9 } from '../../shared/extensionUtilities'
-import { isIamConnection } from '../../auth/connection'
 import { createBuilderIdItem, createSsoItem, createIamItem } from '../../auth/utils'
 
-export const showConnectionPrompt = async () => {
-    // Skip this prompt on C9 because:
-    // * The UI looks bad with C9's style of pickers
-    // * C9 will always start with _some_ form of auth so this prompt is less common
-    if (isCloud9()) {
-        if (isCloud9('classic')) {
-            const iamConn = (await AuthUtil.instance.auth.listConnections()).find(isIamConnection)
-            if (iamConn) {
-                await AuthUtil.instance.auth.useConnection(iamConn)
-            }
-        } else {
-            await AuthUtil.instance.connectToAwsBuilderId()
-        }
-
-        return
-    }
+export const showCodeWhispererConnectionPrompt = async () => {
+    const items = isCloud9('classic')
+        ? [createSsoItem(), createCodeWhispererIamItem()]
+        : [createBuilderIdItem(), createSsoItem(), createCodeWhispererIamItem()]
```

Comment:
Should show builder id for cloud9-codecatalyst? https://github.com/aws/aws-toolkit-vscode/pull/4058/commits/661549f62b0142fc5de525bdf977b2ed609579e3

Reply:
This specifies `isCloud9('classic')` so it should show Builder ID for this when C9-codecatalyst. Am I misunderstanding `isCloud()`?

Reply:
here is what i was thinking: https://github.com/aws/aws-toolkit-vscode/commit/661549f62b0142fc5de525bdf977b2ed609579e3

Reply:
oh i missed that, the old code had `isCloud9()` but yeah this makes sense.